src/mlpack/methods/kde/kde_model_impl.hpp

```diff
@@ -61,6 +81,11 @@ inline KDEModel::KDEModel(KDEModel&& other) :
   other.absError = 0;
   other.kernelType = KernelTypes::GAUSSIAN_KERNEL;
   other.treeType = TreeTypes::KD_TREE;
+  other.monteCarlo = false;
+  other.mcProb = 0.95;
+  other.initialSampleSize = 300;
+  other.mcEntryCoef = 3;
+  other.mcBreakCoef = 0.7;
```

Comment:
One disadvantage here is that this code is duplicated, and so if the defaults are ever changed in `KDE`, they won't be changed here unless someone remembers to do that.

Reply:
Solved in 8147e8c33e59d55726ba25a6725722d02e917a75 :)

Reply:
Clever solution :)